test/unit/test_sap_adt_annotation.py

```
@@ -4,7 +4,8 @@
 
 import sap.adt
 from sap.adt.annotations import xml_attribute, xml_element, XmlElementKind, XmlNodeProperty, XmlElementProperty, \
-                                XmlAttributeProperty, XmlNodeAttributeProperty, XmlContainer, XmlListNodeProperty
+                                XmlAttributeProperty, XmlNodeAttributeProperty, XmlContainer, XmlListNodeProperty, \
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)